automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -63,12 +88,35 @@ object Leonardo extends RestClient with LazyLogging {
       handleClusterSeqResponse(parseResponse(getRequest(s"$url/$path")))
     }
 
+    def listIncludingDeletedRuntime(googleProject: GoogleProject)(implicit token: AuthToken): Seq[ListRuntimeResponseCopy] = {
+      val path = s"api/google/v1/runtimes/${googleProject.value}?includeDeleted=true"
+      logger.info(s"Listing runtimes including deleted in project: GET /$path")
+      val parsedRequest = parseResponse(getRequest(s"$url/$path"))
+      //logger.info(s"PARSED RESPONSE ${parsedRequest}")
+      //Something going wrong here
+      handleListRuntimeResponse(parseResponse(getRequest(s"$url/$path")))
+    }
+
     def create(googleProject: GoogleProject, clusterName: RuntimeName, clusterRequest: ClusterRequest)(
       implicit token: AuthToken
     ): ClusterCopy = {
+
+
       val path = clusterPath(googleProject, clusterName, Some(ApiVersion.V2))
       logger.info(s"Create cluster: PUT /$path")
       handleClusterResponse(putRequest(url + path, clusterRequest))
+
+    }
+
+    def createRuntime(googleProject: GoogleProject, clusterName: RuntimeName, clusterRequest: RuntimeRequest)(
+      implicit token: AuthToken
+    ): Unit = {
+
+      val path = runtimePath(googleProject, clusterName, Some(ApiVersion.V1))
+      logger.info(s"Create runtime: POST /$path")
```

Comment:
is the log necessary?

Reply:
It seems like this log gets used in the other ____Runtime methods. So I have just left it in.  I can also remove it if needed though. It's not essential.